lib/landing_session.js

```diff
@@ -306,9 +306,24 @@ export default class LandingSession extends Session {
     const original = runSync('git', [
       'show', 'HEAD', '-s', '--format=%B'
     ]).trim();
+
+    // git has very specific rules about what is a trailer and what is not.
+    // Instead of trying to implement those ourselves, let git parse the
+    // original commit message and see if it outputs any trailers.
+    const originalHasTrailers = runSync('git', [
+      'interpret-trailers', '--parse', '--no-divider'
+    ], {
+      input: `${original}\n`
+    }).trim().length !== 0;
+
     const metadata = this.metadata.trim().split('\n');
     const amended = original.split('\n');
-    if (amended[amended.length - 1] !== '') {
```

Comment:
This condition appears to have always been `false` because the code above calls `trim()` on the commit message before splitting it at `\n`.